src/features/search/index.tsx

```diff
@@ -74,36 +68,35 @@ export const Search: FC = () => {
     const search: Search = useCallback((query: string) => dispatch(updateTicker(query)), [query, dispatch]);
     const errorQuote = ''
 
+    const stockList = useSelector((state: AppState) => state.stockList)
+
     useEffect(() => {
         if(errorQuote.length > 0) {
```

Comment:
Q: what's `errorQuote` meant to be/do? It's hard to review on my phone properly, but looks like this `if` block will never run, as `errorQuote` is ALWAYS an empty string.

Reply:
the error handling was being worked on as I forked off from egills repo and I haven't had the time to get round to actually fixing it.

Reply:
:+1: Could you make an issue to track this feature, please?

Reply:
yes sorry i thought i did. done now!